4 - A quadratic equation/solution_4.rb

```diff
@@ -0,0 +1,39 @@
+MULTIPLY_FIRST_COEFFICIENT = 2
+DISCRIMINANT_FORMULA_NUMBER = 4
+
+class Quad
+  def initialize(first_coefficient, second_coefficient, third_coefficient)
+    @first_coefficient = first_coefficient.to_f
+    @second_coefficient = second_coefficient.to_f
+    @third_coefficient = third_coefficient.to_f
+    @discriminant = get_discriminant
+  end
+
+  def get_discriminant
+    @second_coefficient * @second_coefficient - DISCRIMINANT_FORMULA_NUMBER * @first_coefficient * @third_coefficient
```

Comment:
please use parentheses to avoid errors in the sequence of mathematical functions

Reply:
@alexshvedko added parentheses